grakn-core/src/main/java/ai/grakn/util/REST.java

```diff
@@ -104,15 +104,14 @@
         public static final String ACTION_QUERY_ABORT = "queryAbort";
         public static final String ACTION_COMMIT = "commit";
         public static final String ACTION_ROLLBACK = "rollback";
-        public static final String ACTION_AUTOCOMPLETE = "autocomplete";
         public static final String ACTION_PING = "ping";
+        public static final String ACTION_TYPES = "types";
```

Comment:
Yes, this one is a possible value for the JSON key "action", the other is a JSON key itself.